pkg/deployment/resources/secret_hashes.go

```diff
@@ -78,6 +87,18 @@ func (r *Resources) ValidateSecretHashes() error {
 				Str("expected-hash", expectedHash).
 				Str("new-hash", hash).
 				Msg("Secret has changed.")
+			if actionHashChanged != nil {
+				if err := actionHashChanged(r.context, secret); err != nil {
+					log.Debug().Msg("Failed to change secret hash")
```

Comment:
better do a `failed to change secret: hash-changed-action returned error: ...`

Reply:
changed